app/helpers/games_helper.rb

```diff
@@ -1,2 +1,13 @@
 module GamesHelper
+  def render_piece(x, y)
+    pieces = Piece.all
+    puts pieces.size
+    pieces.each do |piece|
+      puts piece.type
+      if piece.x_position == x && piece.y_position == y
+        return "#{piece.type} #{piece.color}"
+      end
+    end
```

Comment:
This would work. A more efficient way of finding the piece would be to let Postgres do it for you.

`@game.pieces.find_by(x_position: x, y_position: y)`. This will either return a piece, or be nil.